Source/WebCore/html/ImageBitmap.cpp

```diff
@@ -108,15 +108,25 @@ RefPtr<ImageBuffer> ImageBitmap::createImageBuffer(ScriptExecutionContext& scrip
     return ImageBuffer::create(size, RenderingPurpose::Canvas, resolutionScale, *imageBufferColorSpace, PixelFormat::BGRA8, bufferOptions, { client });
 }
 
-void ImageBitmap::createPromise(ScriptExecutionContext& scriptExecutionContext, ImageBitmap::Source&& source, ImageBitmapOptions&& options, ImageBitmap::Promise&& promise)
+void ImageBitmap::createCompletionHandler(ScriptExecutionContext& scriptExecutionContext, ImageBitmap::Source&& source, ImageBitmapOptions&& options, ImageBitmapCompletionHandler&& completionHandler)
 {
-    WTF::switchOn(source,
+    return WTF::switchOn(source,
```

Comment:
Why do you return here? The method is still void.

Reply:
I don't know if we have a style rule regarding this, but the advantage to returning here is to ensure the result of switchOn (void, in this case) and the result of this member function (void) match. Alternatively unused result warning would catch that, I'm not sure if we build with that.